pinot-query-planner/src/test/java/org/apache/pinot/query/testutils/MockRoutingManagerFactory.java

```diff
@@ -178,6 +180,12 @@ public TimeBoundaryInfo getTimeBoundaryInfo(String tableName) {
           String.valueOf(System.currentTimeMillis() - TimeUnit.DAYS.toMillis(1))) : null;
     }
 
+    @Nullable
+    @Override
+    public TablePartitionInfo getTablePartitionInfo(String tableNameWithType) {
+      return null;
+    }
```

Comment:
chatted with @Jackie-Jiang offline, we should not mock the TableDataManager/InstanceDataManager going forward, otherwise whenever the 2 main file changes we need to make sure the mock behavior aligns with the change. 

instead we should start them normally (TODO/follow up)